client/src/app/managing-body-detail/managing-body-detail.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'op-managing-body-detail',
```

Comment:
In the issue there is a mission to delete all selector's that not in use,
so i checked for one in the html template's for this component and there isn't any.
Thought that this is maybe a misunderstanding of mine.
revert this change?